docker/Dockerfile-ca

```diff
@@ -1,47 +1,49 @@
-# This dockerfile builds a container capable of running the SSH CA bot. Note that a lot of this code is duplicated
-# between this file and Dockerfile-kssh.
-FROM ubuntu:18.04
-
-# Dependencies
-RUN apt-get -qq update
-RUN apt-get -qq  install curl software-properties-common ca-certificates gnupg -y
-RUN useradd -ms /bin/bash keybase
-USER keybase
+FROM alpine:latest AS builder
+
+# add dependencies
+RUN apk update && apk add --no-cache \
+    go curl git musl-dev
+
+# build keybase binary
+WORKDIR /go
+ENV GOPATH=/go
+ENV KEYBASE_VERSION=5.0.0
```

Comment:
imho pinning is better practice, to prevent unexpected behavior from changes in dependencies depending on when the docker image was built. plus, not pinning here would mean that changes in dependencies don't trigger any CI/testing.

i think if keybase provides a dockerfile that is guaranteed to work as expected (under most cases -- barring deprecation), it's up to the user to continue to pull from master (where we update dep versions, and test the updates, as often as we can -- in the future moving towards a more automated release process) and do any other version updates that they want. and hopefully they upstream their fixes if something on master actually deprecates :)